src/openrct2/management/NewsItem.h

```diff
@@ -315,4 +321,7 @@ namespace OpenRCT2::News
 
     void AddItemToQueue(News::Item* newNewsItem);
     void RemoveItem(int32_t index);
+
+    void importNewsItems(
+        GameState_t& gameState, const std::vector<News::Item>& recent, const std::vector<News::Item>& archived);
```

Comment:
The implementation being a vector isn't really relevant here, right. I'd suggest taking the C++20 approach of using std::span for the parameters. The construction is implicit on the calling side. (Just need to swap out the include in this header.)